packages/x-charts/src/PieChart/PieArc.tsx

```diff
@@ -86,8 +86,8 @@ export type PieArcProps = Omit<React.SVGProps<SVGPathElement>, 'ref' | 'id'> &
     outerRadius: number;
     paddingAngle: number;
     startAngle: number;
-    /** @default false */
-    skipAnimation: boolean;
+    skipAnimation?: boolean;
+    skipInteraction?: boolean;
```

Comment:
`PieArc` is exported, do we want to document these properties? At least `skipInteraction`.